doc/extensibility/IVetoProjectLoad.md

```diff
@@ -3,4 +3,24 @@
 
 **[Item template:](project_item_templates.md)** Project Load Veto extension
 
-TODO
+To veto a project load there is two extension points `IVetoProjectLoad` and `IVetoProjectPreLoad`.
+The difference is that `IVetoProjectPreLoad` is called with the configuration about to be loaded.
+
+Example implementation:
```

Comment:
I think the code is self explanatory. `AllowProjectLoadAsync` is a very descriptive name.